mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubResponseParser.kt

```diff
@@ -32,12 +32,16 @@ import io.spine.json.Json
 
 /**
  * Parses `IssuesAndPullRequestsSearchResult` from the JSON.
+ *
+ * @param json The string containing JSON with the `IssuesAndPullRequestsSearchResult`.
  */
 public fun parseIssuesAndPullRequestsFromJson(json: String): IssuesAndPullRequestsSearchResult =
```

Comment:
Can't we instead create these as extensions to the `KClass`-es of the respective types? That would allow us to shorten the extension names to `parseJson(...)`.